### PR TITLE
New Instrument Cal CSVs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by snwhite** [comment]
[PYIV Instrument Cal CSVs [DT-158]](https://trello.com/c/I1InRCiO/158-pyiv-instrument-cal-csvs-dt-158)

File: calibration/PCO2WC/CGINS-PCO2WC-C0107__20211228.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0107,CC_cala,-0.0006, source_file PCO2W-C_SAMI-CO2_SN_C0107_Calibration_Files_2021-12-28.zip
+C0107,CC_calb,0.8945,
+C0107,CC_calc,-1.9528,
+C0107,CC_calt,14.5373,
+C0107,CC_ea434,19706,constant
+C0107,CC_ea620,34,constant
+C0107,CC_eb434,3073,constant
+C0107,CC_eb620,44327,constant
```
Comment (sbeaulieu) [inline calibration/PCO2WC/CGINS-PCO2WC-C0107__20211228.csv:9]: OK

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20211231.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...9, 272.82, 273.64, 274.46, 275.28, 276.1, 276.93, 277.75, 278.57, 279.39, 280.21, 281.04, 281.86, 282.68, 283.5, 284.32, 285.15, 285.97, 286.79, 287.61, 288.43, 289.26, 290.08, 290.9, 291.72, 292.54, 293.37, 294.19, 295.01, 295.83, 296.65, 297.47, 298.3, 299.12, 299.94, 300.76, 301.58, 302.4, 303.22, 304.05, 304.87, 305.69, 306.51, 307.33, 308.15, 308.97, 309.79, 310.61, 311.43, 312.26, 313.08, 313.9, 314.72, 315.54, 316.36, 317.18, 318, 318.82, 319.64, 320.46, 321.28, 322.1, 322.92, 323.74, 324.55, 325.37, 326.19, 327.01, 327.83, 328.65, 329.47, 330.29, 331.1, 331.92, 332.74, 333.56, 334.38, 335.19, 336.01, 336.83, 337.65, 338.46, 339.28, 340.1, 340.92, 341.73, 342.55, 343.37, 344.18, 345, 345.81, 346.63, 347.44, 348.26, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.78, 355.59, 356.41, 357.22, 358.03, 358.85, 359.66, 360.47, 361.28, 362.1, 362.91, 363.72, 364.53, 365.34, 366.16, 366.97, 367.78, 368.59, 369.4, 370.21, 371.02, 371.83, 372.64, 373.45, 374.26, 375.07, 375.88, 376.69, 377.49, 378.3, 379.11, 379.92, 380.73, 381.53, 382.34, 383.15, 383.95, 384.76, 385.57, 386.37, 387.18, 387.98, 388.79, 389.59, 390.4, 391.2, 392.01, 392.81, 393.61, 394.42, 395.22, 396.02]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20211231.csv:8]: OK

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20220104.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
..., 271.85, 272.66, 273.47, 274.27, 275.08, 275.89, 276.7, 277.51, 278.32, 279.13, 279.94, 280.75, 281.56, 282.37, 283.18, 283.99, 284.8, 285.61, 286.42, 287.23, 288.03, 288.84, 289.65, 290.46, 291.27, 292.08, 292.89, 293.7, 294.51, 295.32, 296.12, 296.93, 297.74, 298.55, 299.36, 300.17, 300.98, 301.78, 302.59, 303.4, 304.21, 305.02, 305.82, 306.63, 307.44, 308.25, 309.06, 309.86, 310.67, 311.48, 312.29, 313.09, 313.9, 314.71, 315.51, 316.32, 317.13, 317.93, 318.74, 319.55, 320.35, 321.16, 321.97, 322.77, 323.58, 324.38, 325.19, 326, 326.8, 327.61, 328.41, 329.22, 330.02, 330.83, 331.63, 332.44, 333.24, 334.04, 334.85, 335.65, 336.46, 337.26, 338.06, 338.87, 339.67, 340.47, 341.28, 342.08, 342.88, 343.68, 344.49, 345.29, 346.09, 346.89, 347.69, 348.5, 349.3, 350.1, 350.9, 351.7, 352.5, 353.3, 354.1, 354.9, 355.7, 356.5, 357.3, 358.1, 358.9, 359.7, 360.5, 361.29, 362.09, 362.89, 363.69, 364.49, 365.28, 366.08, 366.88, 367.67, 368.47, 369.27, 370.06, 370.86, 371.66, 372.45, 373.25, 374.04, 374.84, 375.63, 376.42, 377.22, 378.01, 378.81, 379.6, 380.39, 381.19, 381.98, 382.77, 383.56, 384.35, 385.15, 385.94, 386.73, 387.52, 388.31, 389.1, 389.89, 390.68, 391.47, 392.26, 393.05, 393.84]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01094__20220104.csv:8]: OK

File: calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1218,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1218_Calibration_Files_2022-01-13.zip > BBFL2W-1218.dev
```
Comment (sbeaulieu) [inline calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv:2]: the source filename extension is capitalized in Vault BBFL2W-1218.DEV

File: calibration/CTDBPP/CGINS-CTDBPP-50072__20210617.csv [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+16-50072,CC_pa2,2.193415e-11,
+16-50072,CC_ptca0,5.240239e+05,
+16-50072,CC_ptca1,-1.732149e+01,
+16-50072,CC_ptca2,2.534939e-01,
+16-50072,CC_ptcb0,2.498613e+01,
+16-50072,CC_ptcb1,-3.750000e-04,
+16-50072,CC_ptcb2,0.000000e+00,
+16-50072,CC_ptempa0,-1.170859e+02,
+16-50072,CC_ptempa1,1.108305e+02,
+16-50072,CC_ptempa2,-1.798709e+01,
```
Comment (sbeaulieu) [inline calibration/CTDBPP/CGINS-CTDBPP-50072__20210617.csv:23]: OK

Comment (cdobs) [inline calibration/CTDBPP/CGINS-CTDBPP-50072__20210617.csv:23]: looks good

File: calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+1218,CC_dark_counts_cdom,50,
+1218,CC_dark_counts_chlorophyll_a,55,
+1218,CC_dark_counts_volume_scatter,56,
+1218,CC_depolarization_ratio,0.039,Constant
+1218,CC_measurement_wavelength,700,Constant
+1218,CC_scale_factor_cdom,0.0727,
+1218,CC_scale_factor_chlorophyll_a,0.0110,
+1218,CC_scale_factor_volume_scatter,1.863e-06,
+1218,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv:11]: looks good

[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv:11]: agreed

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20211231.csv (hunk 2 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20211231.csv:8]: spot check, looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20220104.csv (hunk 3 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01094__20220104.csv:8]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0107__20211228.csv (hunk 1 above)
Comment (cdobs) [inline calibration/PCO2WC/CGINS-PCO2WC-C0107__20211228.csv:9]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0108__20220103.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0108,CC_cala,0.0642,source_file PCO2W-C_SAMI-CO2_SN_C0108_Calibration_Files_2022-01-03.zip
+C0108,CC_calb,0.5191,
+C0108,CC_calc,-1.3970,
+C0108,CC_calt,14.3639,
+C0108,CC_ea434,19706,constant
+C0108,CC_ea620,34,constant
+C0108,CC_eb434,3073,constant
+C0108,CC_eb620,44327,constant
```
Comment (cdobs) [inline calibration/PCO2WC/CGINS-PCO2WC-C0108__20220103.csv:9]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0120__20211214.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0120,CC_cala,0.0193,source_file PCO2W-C_SAMI-CO2_SN_C0120_Calibration_Files_2021-12-14.zip
+C0120,CC_calb,0.7657,
+C0120,CC_calc,-1.7511,
+C0120,CC_calt,13.7885,
+C0120,CC_ea434,19706,constant
+C0120,CC_ea620,34,constant
+C0120,CC_eb434,3073,constant
+C0120,CC_eb620,44327,constant
```
Comment (cdobs) [inline calibration/PCO2WC/CGINS-PCO2WC-C0120__20211214.csv:9]: looks good

**Review by cdobs — APPROVED** [review]
looks good. FLORTD file extension to be capitalized?

File: calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv (hunk 4 above)
Comment (rtravis-whoi) [reply calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv:2]: doesn't matter - this is ok

File: calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv (hunk 6 above)
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv:11]: same - extension doesn't matter if it matches w/ capitalization or not.

**Review by sbeaulieu — APPROVED** [review]

**Pull request merged**